static/app/views/replays/deadRageClick/deadRageSelectorCards.tsx

```diff
@@ -1,89 +1,268 @@
-import {Fragment} from 'react';
+import {ComponentProps, Fragment, ReactNode, useMemo, useState} from 'react';
 import styled from '@emotion/styled';
 import {Location} from 'history';
 
-import {IconCursorArrow} from 'sentry/icons';
+import {LinkButton} from 'sentry/components/button';
+import EmptyStateWarning from 'sentry/components/emptyStateWarning';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import TextOverflow from 'sentry/components/textOverflow';
+import {IconCursorArrow, IconWarning} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import {space} from 'sentry/styles/space';
+import EventView from 'sentry/utils/discover/eventView';
+import getRouteStringFromRoutes from 'sentry/utils/getRouteStringFromRoutes';
 import useDeadRageSelectors from 'sentry/utils/replays/hooks/useDeadRageSelectors';
+import useReplayList from 'sentry/utils/replays/hooks/useReplayList';
 import {useLocation} from 'sentry/utils/useLocation';
-import SelectorTable from 'sentry/views/replays/deadRageClick/selectorTable';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useRoutes} from 'sentry/utils/useRoutes';
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
+import Accordion from 'sentry/views/performance/landing/widgets/components/accordion';
+import {RightAlignedCell} from 'sentry/views/performance/landing/widgets/components/selectableList';
+import {
+  ContentContainer,
+  HeaderContainer,
+  HeaderTitleLegend,
+  StatusContainer,
+  Subtitle,
+  WidgetContainer,
+} from 'sentry/views/profiling/landing/styles';
+import {ReplayCell} from 'sentry/views/replays/replayTable/tableCell';
+import {ReplayListLocationQuery} from 'sentry/views/replays/types';
+
+function transformSelectorQuery(selector: string) {
+  return selector
+    .replaceAll('"', `\"`)
+    .replaceAll('aria=', 'aria-label=')
+    .replaceAll('testid=', 'data-test-id=');
+}
+
+function UseExampleReplays(selector, location, clickType, deadOrRage) {
```

Comment:
This component name is funny, it's similar to a react-hook `useFoo()` that it made me stop. How about something like `<ExampleReplaysList>`?

Also, it could go into it's own file, which would just trim this file down a bit making it easier to see the main accordian bits that are in here